app/models/project.rb

```diff
@@ -1,6 +1,15 @@
 class Project < ActiveRecord::Base
   # skip_before_filter :verify_authenticity_token
 
-  belongs_to :user
+  belongs_to :user, :foreign_key => :owner
+
+  has_attached_file :snap_file,
+  :storage => :s3,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.